src/main/java/io/github/railroad/settings/ui/themes/ThemeDownloadManager.java

```diff
@@ -99,11 +129,6 @@ public static List<Theme> fetchThemes(final String url) {
             }
 
             ResponseBody body = response.body();
-            if (body == null) {
-                LOGGER.error("While fetching themes, the body was null");
-                return itemList;
-            }
-
```

Comment:
Is there a reason for removing this? The body being null is a possibility afaik.